WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -534,7 +534,14 @@ def stripe_webhook
     else
       logger.info "Unhandled Stripe event type: #{event.type}"
     end
-
+    if stored_intent.holder.type == "attendee"
```

Comment:
This will likely not work even now, because `.holder` will already return the AttendeeRequest object and that in turn does **not** have a `type` field. Perhaps something like `stored_intent.holder.is_a? AttendeePaymentRequest`?